text_formatter.go

```diff
@@ -78,6 +82,22 @@ func (f *TextFormatter) init(entry *Entry) {
 	}
 }
 
+func (f *TextFormatter) isColored() bool {
+	isColored := f.ForceColors || f.isTerminal
+
+	if f.OverrideColors {
+		if force, ok := os.LookupEnv("CLICOLOR_FORCE"); ok && force != "0" {
+			isColored = true
+		}
+
+		if os.Getenv("CLICOLOR") == "0" {
+			isColored = false
+		}
+	}
+
+	return isColored && !f.DisableColors
```

Comment:
I decided to give maximum priority to `DisableColors` also when `OverrideColors == true` as this is the only way for a developer to completely disable colored output